### PR TITLE
I add in some testing and keys to the text and button, and I added some screen and functionality

### PR DESCRIPTION
**Pull request by JosefADVassell**

### CODE REVIEW

File: lib/rent_a_product/buyaproduct.dart [hunk 1]
```diff
@@ -11,47 +12,77 @@ class BuyProduct extends StatefulWidget {
...
+                    /* ElevatedButton(
+                        onPressed: () {
+                          FirebaseFirestore.instance
+                              .collection("Products")
+                              .doc(widget.productID)
+                              .update({
+                            'is_available': isAvailable,
+                          });
+                        },
+                        child: Text("Rent $value")),*/
```
Comment (LeoRuiz01) [inline lib/rent_a_product/buyaproduct.dart:54]: If this is not going to be used in the future, remove the comments from the file.

**Review by LeoRuiz01 — CHANGES_REQUESTED** [review]
Add comments to  shopping_cart_product.dart, checkout_screen.dart, rentedscreen.dart, buyaproduct (change the name to match the naming schema to match the other files i.e. rented_screen for both rentedscreen and buyproduct.). 

The reason comments are being requested is because this is a 930 line PR that doesn't include an explanation to what is going on. The comments should explain what is going on each file without being too complex. ( The comment should not be longer than 3 lines).

**Review by LeoRuiz01 — APPROVED** [review]
LGTM

**Review by crzaratech — APPROVED** [review]
Nice, but remember to make different PRs for different features. All the added features were good

**Pull request merged**